@navikt/core/react/src/form/combobox/Input/Input.tsx

```diff
@@ -132,17 +149,28 @@ const Input = forwardRef<HTMLInputElement, InputProps>(
           if (activeDecendantId || value) {
             e.preventDefault();
           }
+        } else if (["ArrowLeft", "ArrowRight"].includes(e.key)) {
+          /**
+           * In case user has an active selection and 'completes' the selection with ArrowLeft or ArrowRight
+           * we need to make sure to update the filter.
+           */
+          if (value !== "" && value !== searchTerm) {
+            onChange(value);
+          }
         } else if (e.key === "ArrowDown") {
-          // Check that cursor position is at the end of the input field,
-          // so we don't interfere with text editing
-          if (e.target.selectionStart === value?.length) {
-            e.preventDefault();
-            if (virtualFocus.activeElement === null || !isListOpen) {
-              toggleIsListOpen(true);
-            }
-            virtualFocus.moveFocusDown();
+          // Reset the value to the search term to cancel autocomplete
+          // if the user moves focus down to the FilteredOptions
+          if (internalRef.current?.selectionStart) {
```

Comment:
Do we need this check?

Reply:
I think the purpose was to stop unnecessary calls to setValue, but this does not work, as selectionStart will indicate the position of the cursor if no text is selected. 🤦 

Should we replace it with the following code, or is it harmless to have extra calls to setValue?

```suggestion
          if (value !== "" && value !== searchTerm) {
```

Reply:
Should be harmless, but might be nice to have the `if` for clearity. Do we need to check that `value` is not empty though?

Reply:
Good point!
```suggestion
          if (value !== searchTerm) {
```